.circleci/config.yml

```diff
@@ -4,23 +4,57 @@
 version: 2
 jobs:
   build:
-    docker:
-      - image: circleci/golang:1.11
-    working_directory: /go/src/github.com/litmuschaos/chaos-operator
+    machine:
+        image: circleci/classic:201808-01
+        docker_layer_caching: true
+    environment:
+      K8S_VERSION: v1.12.0
+      KUBECONFIG: /home/circleci/.kube/config
+      MINIKUBE_VERSION: v1.3.1
+      MINIKUBE_WANTUPDATENOTIFICATION: false
+      MINIKUBE_WANTREPORTERRORPROMPT: false
+      MINIKUBE_HOME: /home/circleci
+      CHANGE_MINIKUBE_NONE_USER: true 
+
+    working_directory: ~/go/src/github.com/litmuschaos/chaos-operator
+    
     steps:
       - checkout
+      - run:
+         name: Setup kubectl
+         command: | 
+           curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+      - run:
+          name: Setup minikube
+          command: |
+            curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64  && chmod +x minikube
```

Comment:
Instead of installing the latest minikube version can use some specific version !!